### DISCUSSION TITLE
Feedback - Peppy

### DISCUSSION THREAD

**Discussion by peppy10003** [WillTheDeveloper/StudyPortal#14] [General]
Thoughts on design:
- Nice simple design
- having a way to teach new users about the system might be a good idea, such as a pop up for new users that explains what each part of the UI does (a small animation in a box might work).
Thoughts on overall idea:
- A clear example of use may be helpful.
like or dislike:
the idea is pretty good however, the lack of clear uses and examples of scenarios for its uses makes it hard to give a concrete decision.

**Comment by WillTheDeveloper** [comment]
Thanks for your feedback! @peppy10003 I am glad that you like the idea and see the value it would have for students.

I will take your feedback into consideration and will look at improving the README.md file which contains everything about the project.

**Comment by WillTheDeveloper** [comment]
> such as a pop up for new users that explains what each part of the UI does (a small animation in a box might work)

Regarding this comment, do you believe that it would help you and other students that are using the platform? @peppy10003

**Reply by peppy10003** [reply]
yes
